fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -1504,7 +1504,7 @@ public String getConnectorSinkCompressionCodec() {
     private boolean enableMaterializedViewRewritePartitionCompensate = true;
 
     @VarAttr(name = ENABLE_FORCE_RULE_BASED_MV_REWRITE)
-    private boolean enableForceRuleBasedMvRewrite = false;
+    private boolean enableForceRuleBasedMvRewrite = true;
 
```

Comment:
Will this make the rewrite strategy too aggressive?

Reply:
Just like `pruneTables`, we can do materialized view rewrite in RBO which will make mv rewrite easier and more explainable, and we also keep the CBO.

There may be some bad cases:
1. increase optimizer time
2. mv rewrite is bad than no rewrite

But I think it's better to make mv rewrite more stable than those since those are rare, and we can fix those later if we met.